src/Private/Responses.ps1

```diff
@@ -48,4 +82,429 @@ function Show-PodeErrorPage {
+        }
+
+        # Determine if the server is running in Windows mode or is running a varsion that support Linux
+        # https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-childitem?view=powershell-7.4#example-10-output-for-non-windows-operating-systems
+        $windowsMode = ((Test-PodeIsWindows) -or ($PSVersionTable.PSVersion -lt [version]'7.1.0') )
+
+        # Construct the HTML content for the file browser view
+        $htmlContent = [System.Text.StringBuilder]::new()
+
+        $atoms = $WebEvent.Path -split '/'
+        $atoms = @(foreach ($atom in $atoms) {
+                if (![string]::IsNullOrEmpty($atom)) {
+                    [uri]::EscapeDataString($atom)
+                }
+            })
+        if ([string]::IsNullOrWhiteSpace($atoms)) {
+            $baseLink = ''
+        }
+        else {
+            $baseLink = "/$($atoms -join '/')"
+        }
+
+        # Handle navigation to the parent directory (..)
+        if ($leaf -ne '/') {
+            $LastSlash = $baseLink.LastIndexOf('/')
+            if ($LastSlash -eq -1) {
+                Set-PodeResponseStatus -Code 404
+                return
+            }
+            $ParentLink = $baseLink.Substring(0, $LastSlash)
+            if ([string]::IsNullOrWhiteSpace($ParentLink)) {
+                $ParentLink = '/'
+            }
+            $item = Get-Item '..'
+            if ($windowsMode) {
+                $htmlContent.Append("<tr> <td class='mode'>")
+                $htmlContent.Append($item.Mode)
+            }
+            else {
+                $htmlContent.Append("<tr> <td class='unixMode'>")
+                $htmlContent.Append($item.UnixMode)
+                $htmlContent.Append("</td> <td class='user'>")
+                $htmlContent.Append($item.User)
+                $htmlContent.Append("</td> <td class='group'>")
+                $htmlContent.Append($item.Group)
+            }
+            $htmlContent.Append("</td> <td class='dateTime'>")
+            $htmlContent.Append($item.CreationTime.ToString('yyyy-MM-dd HH:mm:ss'))
+            $htmlContent.Append("</td> <td class='dateTime'>")
+            $htmlContent.Append($item.LastWriteTime.ToString('yyyy-MM-dd HH:mm:ss'))
+            $htmlContent.Append( "</td> <td class='size'></td> <td class='icon'><i class='bi bi-folder2-open'></td> <td class='name'><a href='")
+            $htmlContent.Append($ParentLink)
+            $htmlContent.AppendLine("'>..</a></td> </tr>")
+        }
+        # Retrieve the child items of the specified directory
+        $child = Get-ChildItem -Path $Path -Force
+        foreach ($item in $child) {
+            $link = "$baseLink/$([uri]::EscapeDataString($item.Name))"
+            if ($item.PSIsContainer) {
+                $size = ''
+                $icon = 'bi bi-folder2'
+            }
+            else {
+                $size = '{0:N2}KB' -f ($item.Length / 1KB)
+                $icon = 'bi bi-file'
+            }
```

Comment:
Change `$icon` to be emojis, this way we can remove the dependency on bootstrap - I also think it looks nicer as well 🤣 
```powershell
if ($item.PSIsContainer) {
    $size = ''
    $icon = '📁'
}
else {
    $size = '{0:N2}KB' -f ($item.Length / 1KB)
    $icon = '📄'
}
```

Reply:
I was not aware that something like that will work :) 
Let me try

Reply:
I love it